.travis.yml

```
@@ -0,0 +1,22 @@
+language: swift
+
+osx_image: xcode10.3
+
+cache:
+  bundler: true
+  cocoapods: true
+
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+  - WORKSPACE=hackYourFuture.xcworkspace
+  - SCHEME="hackYourFuture"
+
+before_install:
+  - bundle install
+  - bundle exec pod install
+
+script:
+  - xcodebuild clean -workspace "$WORKSPACE" -scheme "$SCHEME"
+  - xcodebuild  -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone X,OS=12.2' test
```

```suggestion
  - xcodebuild  -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone X,OS=12.4' test
```
On pourrait lancer les tests sur la dernière version iOS ?